submissions/dummy_submission/the-nuclear-options-v1.0.md

```diff
@@ -0,0 +1,233 @@
+ 
+
+# **The Nuclear Options**
+
+<!--Draft 1.0, by Sir Fendi. Images are missing and some uncertainties about UPS are being researched. Also not sure about the appendix. All feedback is welcome about content, layout, or anything else. -->
+
+When I came across a glowing green patch of Uranium ore for the first time, I saw that Factorio had some juicy stuff waiting for me somewhere down the tech tree: Nuclear energy! While in today’s world the trials and triumphs of nuclear power are hotly debated, the Factorio community similarly has its ongoing discussions on whether it is worthwhile: Going nuclear is a couple steps up in complexity from the other power options, while also it is completely optional on the path to launching a rocket. Hence many players tend to not bother with diving into it, or just find a good blueprint as a blackbox and call it a day. Others hear of the warnings of the nuclear reactor as a UPS killer, or even a literal killer when it explode dramatically, and they decide to steer clear of it. Yes, there is a lot to look out for, but you can trust in Wube: Like many other carefully balanced mechanics of the game, nuclear power is reasonably easy to get into and you can quickly make something that just works. As for the more complicated and colorful part, I have a few guidelines to offer from my own fair share of tinkering and the findings of the glowing gems of our community.
+

```

Comment:
I always push for articles to have more media, because it was one of my favourite aspects of FFF. Maybe here we could add a gif of a uranium patch glowing at night. That always looks beautiful :)

Reply:
I have not given proper thought to the media side yet. This one definitely! I will do a pass-through for other suitable places.